blocks/core/ff_container/ff_container-dialog/ff_container-dialog.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+var React = require('react');
+
+function generateClass(base, props){
+    var classNames = [];
+        classNames.push(base);
+        if (!!props.modifier) classNames.push(base + '--' + props.modifier);
```

Comment:
I think the distinction should only matter when testing equality. For example later on you've used `switch(true)` which is going to test equality with `true` rather than testing truthiness, but an `if` will test truthiness so you should be alright without it.